library/x509_create.c

```diff
@@ -116,16 +117,21 @@ int mbedtls_x509_string_to_names( mbedtls_asn1_named_data **head, const char *na
 
         if( !in_tag && *c == '\\' && c != end )
```

Comment:
agreed on first coment, on second comment, we rely on `c == end` so we would store the named data, in line 134